massa-proto/proto/massa/api/v1/versioning.proto

```diff
@@ -46,3 +46,10 @@ enum ComponentStateId {
   COMPONENT_STATE_ID_ACTIVE = 5;
   COMPONENT_STATE_ID_FAILED = 6;
 }
+
+// Same as MipComponent enum in versioning package
```

Comment:
In proto files, always copy/paste the doc on Rust struct. This is will be added to autogenerated documentation

Reply:
Updated